integration/spark-datasource/src/test/scala/org/apache/spark/sql/carbondata/datasource/TestCreateTableUsingSparkCarbonFileFormat.scala

```diff
@@ -502,7 +504,9 @@ class TestCreateTableUsingSparkCarbonFileFormat extends FunSuite with BeforeAndA
     checkAnswer(spark.sql("SELECT COUNT(*) FROM (select address,age,note from sdkOutputTableWithoutSchema where length(address)=75000 and length(note)=75000)"),
       Seq(Row(totalRecordsNum)))
     spark.sql("DROP TABLE sdkOutputTableWithoutSchema")
-    clearDataMapCache
+    val mapSize=DataMapStoreManager.getInstance().getAllDataMaps.size()
+    if(mapSize>0)
+      clearDataMapCache
```

Comment:
please remove duplicate code.
please check if this code can be added inside "clearDataMapCache" method